src/FSharpPlus/Extensions.fs

```diff
@@ -370,19 +372,20 @@ module Map =
 
     let unzip (source: Map<'Key, 'T1 * 'T2>) = mapValues fst source, mapValues snd source
 
-    /// Returns the union of two dictionaries, using the combiner function for duplicate keys.
+    /// Returns the union of two maps, using the combiner function for duplicate keys.
     let unionWith combiner (source1: Map<'Key, 'Value>) (source2: Map<'Key, 'Value>) =
         Map.fold (fun m k v' -> Map.add k (match Map.tryFind k m with Some v -> combiner v v' | None -> v') m) source1 source2
 
     /// Returns the union of two maps, preferring values from the first in case of duplicate keys.
-    let union (source: Map<'Key, 'T>) (altSource: Map<'Key, 'T>) = unionWith (fun k _ -> k) source altSource
-
+    let union (source: Map<'Key, 'T>) (altSource: Map<'Key, 'T>) = 
```

Comment:
If this is a reversion, could you leave exactly as it was?

This is mainly to reduce the signal to noise ratio of the changes?

Reply:
done.